src/systems/filecoin_vm/interpreter/vm_interpreter.go

```diff
@@ -119,21 +134,27 @@ func (vmi *VMInterpreter_I) ApplyMessage(inTree st.StateTree, message msg.Unsign
 
 	outTree, err = outTree.Impl().WithIncrementedCallSeqNum(message.To())
 	if err != nil {
-		// TODO: if actor deletion is possible at some point, may need to allow this case
+		// Note: if actor deletion is possible at some point, may need to allow this case
 		panic("Internal interpreter error: failed to increment call sequence number")
 	}
 
 	// reward miner gas fees
+
+	TODO() // TODO: Is the burnt funds actor allowed to send money, or only to receive?
```

Comment:
Burnt funds actor should be a sink for now afaik but we might add functionality to it in the future.

Reply:
Sounds good, just wanted to confirm. Will remove the comment and the TODO.